rails-questions.md

```diff
@@ -2,33 +2,51 @@
 
 1. What is the command to create a new Rails application with a Postgres database?
 
+rails new app_name -d postgresql
```

Comment:
Totally. Additionally, you might want to add the flag -T, to drop the default testing environment.